app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User
+
   include Mongoid::Document
   include Mongoid::Userstamp::User
 
```

Comment:
Chuck a `after_update :send_password_change_email, if: :needs_password_change_email?` in here (per https://github.com/plataformatec/devise/wiki/Notify-users-via-email-when-their-passwords-change ) and this should be in business!